react/ContactsListModal/index.jsx

```diff
@@ -61,6 +64,77 @@ const ContactsListModal = props => {
     rest.dismissAction()
   }
 
+  const loading =
+    (contacts.fetchStatus === 'loading' ||
+      contacts.fetchStatus === 'pending') &&
+    !contacts.lastFetch
+
+  const filteredContacts = filterContacts(contacts.data)
+
+  useEffect(() => {
+    const refreshContacts = () => {
+      contacts.fetch()
+    }
+
+    const subscribeRealtime = async () => {
+      try {
+        await client.plugins.realtime.subscribe(
```

Comment:
Should we test if client.plugins.realtime exists and if no tell the dev to register the plugin to get a full featured component?

Reply:
I added it, but thought it was becoming a lot of logic inside the component. So I extracted all of this in a hook. See https://github.com/cozy/cozy-ui/pull/1326/commits/fab16305aac90a08418866b2481764f882179f15